godu.go

```diff
@@ -16,6 +16,7 @@ import (
 
 func main() {
 	limit := flag.Int64("l", 10, "show only files larger than limit (in MB)")
+	nullTerminate := flag.Bool("print0", false, "print null-terminated strings")
```

Comment:
I'm not sure about the reason for this change. Could you please elaborate @leonklingele ?

Reply:
@viktomas see [my concerns](https://github.com/viktomas/godu/issues/48#issuecomment-363584032) and [@Habbie's suggestion](https://github.com/viktomas/godu/issues/48#issuecomment-372641797) on how to circumvent them. Also take a look at the commit messages :)

Reply:
Makes sense. is `-print0` standardised flag in unix commands? If not I'd maybe put longer explanation to the comment e.g. "print null-terminated strings to prevent special characters to affect programs like xargs"

Reply:
__tl;dr__: Instead of separating file names by a line break (`\n`) we should use a null-char delimiter as file names itself can include line breaks.

Reply:
`-print0` is used by `find` for the exact same reason — that's where I got the inspiration from.

Reply:
👍 ok, in that case there are no changes necessary

Reply:
(except for the one test for quote escaping)